cli/lib/kontena/cli/nodes/labels/add_command.rb

```diff
@@ -2,17 +2,17 @@ module Kontena::Cli::Nodes::Labels
   class AddCommand < Kontena::Command
     include Kontena::Cli::Common
 
-    parameter "NODE_ID", "Node id"
+    parameter "NODE", "Node"
```

Comment:
The other, perhaps even better option, would be to send the `Kontena-Node-Name` has a websocket request header, so there would never be any need to create `HostNode`s without any name..